src/index.css

```diff
@@ -69,8 +72,45 @@
     -moz-appearance: textfield; /* Firefox */
   }
   input[type='checkbox'] {
-    accent-color: var(--primary);
-  }  
+    -webkit-appearance: none;
+    position: relative;
+    border-radius: 4px;
+  }
+  input[type='checkbox']:not(:checked) {
+    border: 1px solid rgb(255 255 255 / 0.6);
+  }
+  input[type='checkbox']:not(:checked):hover {
+    border-color: rgb(255 255 255 / 0.8);
+  }
+  input[type='checkbox']:checked {
+    border: 0px;
+    color: rgba(22, 22, 22, var(--tw-text-opacity));
+    background-image: linear-gradient(
+      90deg,
+      var(--gradientFirst) 0%,
+      var(--gradientMiddle) 100%
+    );
+  }
+  input[type='checkbox']:checked:hover {
+    filter: brightness(1.2);
+  }
+  input[type='checkbox']::before {
+    display: none;
+    content: '';
+    position: absolute;
+    top: 60%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 65%;
+    aspect-ratio: 1 / 1;
+    pointer-events: none;
+    background-image: url('assets/icons/check.svg');
```

Comment:
It feels like you should create a checkbox component to manage this with a real svg icon instead of using images

Reply:
Created an issue to make the checkbox implementation uniform throughout the app: https://github.com/bancorprotocol/carbon-app/issues/1544